src/style.css

```diff
@@ -0,0 +1,129 @@
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  position: relative;
+  width: 100%;
+  height: 70px;
+  border: 1px solid #e1e7ed;

```

Comment:
let's try to add this border to input for tests

Reply:
I meant adding border directly to input and remove it from form
cause tests don't see it now
<img width="760" alt="Screenshot 2022-05-19 at 13 48 34" src="https://user-images.githubusercontent.com/57598825/169276504-eb79383f-2dc7-4594-8231-3ad2d61810b2.png">